terraform-stacks/shared_modules/lb/main.tf

```
@@ -0,0 +1,37 @@
+terraform {
+  required_version = ">= 1.0"
+  required_providers {
+    oci = {
+      source  = "oracle/oci"
+      version = ">= 6.12.0"
+    }
+  }
+}
+
+resource "oci_load_balancer_load_balancer" "openshift_api_int_lb" {
+  compartment_id             = var.compartment_ocid
+  display_name               = "${var.cluster_name}-openshift_api_int_lb"
+  shape                      = "flexible"
+  subnet_ids                 = !var.is_control_plane_iscsi_type && !var.is_compute_iscsi_type ? [var.op_subnet_private] : [var.op_subnet_private2]
```

In case of iSCSI, set `subnet_private` IP in the LB.
```suggestion
  subnet_ids                 = var.is_control_plane_iscsi_type && var.is_compute_iscsi_type ? [var.op_subnet_private] : [var.op_subnet_private2]
```

---

The issue Manoj was seeking to resolve with this change was resolved by specifying a rendezvous IP inside the private_two subnet when using Agent-based installation with Bare Metal instances.